Gemfile.lock

```diff
@@ -436,6 +436,7 @@ GEM
 PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
+  arm64-darwin-23
```

Comment:
discussed with @kosiakkatrina, including this as my machine wants this to be in the Gemfile and I am having to remove it from commits all the time